analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarAnalysisContextBaseTest.ShouldAnalyzeTree.cs

```diff
@@ -102,8 +102,8 @@ public void ShouldAnalyzeTree_GeneratedFile_ShouldAnalyzeGeneratedProvider_IsCac
         sut.ShouldAnalyzeTree(tree, CSharpGeneratedCodeRecognizer.Instance).Should().BeTrue();
 
         // GetText should be called every time ShouldAnalyzeGenerated is called...
-        additionalText.Verify(x => x.GetText(It.IsAny<CancellationToken>()), Times.Exactly(3));
-        sonarLintXml.ToStringCallCount.Should().Be(1); // ... but we should only try to read the file once
+        additionalText.Verify(x => x.GetText(It.IsAny<CancellationToken>()), Times.Exactly(6)); // TEMPORARILY BUMPED TO 6
+        sonarLintXml.ToStringCallCount.Should().Be(2); // ... but we should only try to read the file once // TEMPORARILY BUMPED TO 2
```

Comment:
This bump will be reverted once the new SonarLintXmlReader will be used to access all properties.